lawnchair/src/app/lawnchair/gestures/handlers/SleepGestureHandler.kt

```diff
@@ -63,12 +63,11 @@ class SleepGestureHandler(context: Context) : GestureHandler(context) {
 }
 
 class SleepMethodRoot(context: Context) : SleepGestureHandler.SleepMethod(context) {
-    private val rootHelperManager = RootHelperManager.INSTANCE.get(context)
 
-    override suspend fun isSupported() = rootHelperManager.isAvailable()
+    override suspend fun isSupported() = Shell.getShell().isRoot
 
     override suspend fun sleep(launcher: LawnchairLauncher) {
-        rootHelperManager.getService().goToSleep()
+        Shell.su("input keyevent 26").exec()
```

Comment:
And we can put this into `RootHelperManager` and wrap it as a suspend fun like:
```kt
    suspend fun exec(command: String) = withContext(Dispatchers.IO) {
        Shell.getShell().newJob().add(command).exec()
    }
```

Reply:
Yes it's fancy, but why are we doing this?, is there any use cases for this that we can use for? or just for future one? if this for future, I think let's keep it minimal what you think? and apply it if there's other uses cases

Reply:
https://github.com/topjohnwu/libsu?tab=readme-ov-file#shell-operations

`exec` is synchronous, `submit` is async.

Reply:
Yes we only need to execute keyevent 26 once and no job other than allowing root permision, since it's use Shell.su it will ask root permission if magisk, if kernelSu just manually allow root permision in kernel app